### PR TITLE
Add additional slurs configuration option. Closes #1464.

### LINKED ISSUE #1464: Add ability for admins to add words to slur filter
### Feature Proposed

It would be nice to give the ability to add words not already included in the slur filter. This could be done either on the admin level or user level settings

### Describe the solution you'd like

Add an option in the settings to add a list of word to the slur filter. The default slur words in the code should be shown in the help text to prevent duplication.

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/97091749-add-ability-for-admins-to-add-words-to-slur-filter?utm_campaign=plugin&utm_content=tracker%2F126011972&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F126011972&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by ba1ash**
Greetings!
Implemented #1464.
Without a test :( I don't know how to test this gracefully. I don't like the idea of config file modification at runtime. Maybe it's okay to fill in `additional_slurs` option by default with some slurs and rely on them in test.
If someone has a suggestion about the proper test implementation I will be happy to implement it :)

### CODE REVIEW

**Comment by dessalines** [comment]
Sry I've been slow on this, I'll get to it shortly.

File: config/config.hjson [hunk 1]
```diff
@@ -90,4 +90,8 @@
 #    # whether or not smtp connections should use tls
 #    use_tls: true
 #  }
+  # additional_slurs:
+  #  '''
+  #  (\bThis\b)|(\bis\b)|(\bsample\b)
+  #  '''
```
Comment (dessalines) [inline config/config.hjson:96]: The heredoc makes sense here *I think*, although hjson might be able to handle everything after the `:` fine.

**Review by dessalines — APPROVED** [review]
Looks good to me, thanks! I'll let nutomic also go over it.

**Review by Nutomic — APPROVED** [review]

**Comment by Nutomic** [comment]
Thanks @ba1ash!

**Pull request merged**